detections/application/cisco_asa___packet_capture_activity.yml

```diff
@@ -0,0 +1,82 @@
+name: Cisco ASA - Packet Capture Activity
+id: 7e9c3f8a-4b2d-4c5e-9a1f-6d8e5b3c2a9f
+version: 1
+date: '2025-11-18'
+author: Nasreddine Bencherchali, Splunk
+status: production
+type: Anomaly
+description: |
+  This analytic detects execution of packet capture commands on Cisco ASA devices via CLI or ASDM.
+  Adversaries may abuse the built-in packet capture functionality to perform network sniffing, intercept credentials transmitted over the network, capture sensitive data in transit, or gather intelligence about network traffic patterns and internal communications. Packet captures can reveal usernames, passwords, session tokens, and confidential business data.
+  The detection monitors for command execution events (message ID 111008 or 111010) containing "capture" commands, which are used to initiate packet capture sessions on specific interfaces or for specific traffic patterns on the ASA device.
```

Comment:
should we add these "message_id" filters in the search since it would be bit more efficient? or the TA is busted?
Same with other detections